tests/expressions/test_parser.py

```diff
@@ -70,7 +70,6 @@ def test_equals_false() -> None:
 def test_is_null() -> None:
     assert IsNull("foo") == parser.parse("foo is null")
     assert IsNull("foo") == parser.parse("foo IS NULL")
-    assert IsNull("foo") == parser.parse("table.foo IS NULL")
```

Comment:
@kevinjqliu Was this one intentional?

Reply:
yep! its associated with this one https://github.com/apache/iceberg-python/blob/dbcf65b4892779efca7362e069edecff7f2bf69f/pyiceberg/expressions/parser.py#L90-L100